.eslintrc.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    tsconfigDir: __dirname,
+    project: "./tsconfig.json",
+    ecmaVersion: 2020,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true
+    }
+  },
+    ignorePatterns: [
+        "node_modules",
+        "**/.eslintrc.js",
+    ],
+    extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/all",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
```

Comment:
Note the order of these, I think the latter overwrites the former entries (if I remember correctly)